conda/core/prefix_data.py

```diff
@@ -46,8 +47,10 @@ def __init__(self, prefix_path):
 
     def load(self):
         self.__prefix_records = {}
-        for meta_file in glob(join(self.prefix_path, 'conda-meta', '*.json')):
-            self._load_single_record(meta_file)
+        _conda_meta_dir = join(self.prefix_path, 'conda-meta')
+        if lexists(_conda_meta_dir):
+            for meta_file in fnmatch.filter(listdir(_conda_meta_dir), '*.json'):
```

Comment:
What an awful name for a function in the standard library.

Reply:
Would normally do this import as `from fnmatch import filter as fnmatch_filter`. Importing each symbol into the global scope for the module avoids one extra lookup.  Not a big deal here though.